### ISSUE TITLE
AccessDenied error during import existing bucket into state

### ISSUE THREAD

**Issue by imcitius** [scaleway/terraform-provider-scaleway#1806]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

❯ terraform -v
Terraform v1.3.7
on linux_amd64
+ provider registry.terraform.io/cloudflare/cloudflare v3.11.0
+ provider registry.terraform.io/gavinbunney/kubectl v1.14.0
+ provider registry.terraform.io/hashicorp/consul v2.17.0
+ provider registry.terraform.io/hashicorp/helm v2.0.2
+ provider registry.terraform.io/hashicorp/kubernetes v2.18.1
+ provider registry.terraform.io/hashicorp/time v0.9.1
+ provider registry.terraform.io/hashicorp/vault v3.13.0
+ provider registry.terraform.io/scaleway/scaleway v2.11.1

### Affected Resource(s)

* scaleway_object_bucket

### Terraform Configuration Files

```hcl
locals {
  buckets = {
    gitlab = {
      description = "Gitlab"
    }

    mattermost = {
      description = "Mattermost"
    }
  }
}

resource "scaleway_object_bucket" "object_buckets" {
  for_each = local.buckets

  name       = "${var.project}-${each.key}"
  project_id = var.scaleway.project_id
}
```

### Debug Output

[gist.github.com](https://gist.github.com/imcitius/7129f2f6d4ba4f0072fa37dd78c2615e)

### Expected Behavior

Object bucket imported into state

### Actual Behavior

Raised error:
```
Error: couldn't read bucket acl: AccessDenied: Access Denied.
      status code: 403, request id: txd22f8bea1f5b45af9a2c2-0063f91f9f, host id: txd22f8bea1f5b45af9a2c2-0063f91f9f
```

### Steps to Reproduce

1. `terraform import`

### Important Factoids

Terraform is run using terragrunt in my case, but this should not affect. Other types of resources was imported OK.

**Comment by quantumsheep** [comment]
Are you sure you have one of the following permissions? `ObjectStorageBucketsRead`, `ObjectStorageFullAccess`, `AllProductsReadOnly`, `AllProductsFullAccess`

**Comment by imcitius** [comment]
Yes, mentioned user has policy attached with single rule, with scope `All current and future projects` and permission set `AllProductsFullAccess `.
Also in belongs to group `Administrators` with the same rule as above, and one additional rule with `Organization/Organization manager` scope/set.

**Comment by imcitius** [comment]
I have many projects in single SCW account, `var.scaleway.project_id` contains particular project name.
Not default one.

And all works during creation of the resources. I refactored TF code, and needed to move resources inside state.
That's caused problem only during import.

**Comment by quantumsheep** [comment]
@imcitius You should be able to import your bucket with the new `region/name@project_id` format 😀

**Comment by imcitius** [comment]
Thanks for the fix!
